### PR TITLE
[remoteopenhab] Optimize filtering of ALIVE events

### PR DESCRIPTION
**Pull request by lolodomo**
Signed-off-by: Laurent Garnier <lg.hc@free.fr>

### CODE REVIEW

File: bundles/org.openhab.binding.remoteopenhab/src/main/java/org/openhab/binding/remoteopenhab/internal/rest/RemoteopenhabRestClient.java [hunk 1]
```diff
@@ -400,17 +400,15 @@ private void onEvent(InboundSseEvent inboundEvent) {
             connected = true;
             listeners.forEach(listener -> listener.onConnected());
         }
+        if (!"message".equals(name)) {
+            if (!"alive".equals(name)) {
```
Comment (digitaldan) [inline bundles/org.openhab.binding.remoteopenhab/src/main/java/org/openhab/binding/remoteopenhab/internal/rest/RemoteopenhabRestClient.java:404]: Any reason you can't silently ignore anything other than "message"?  Seems like SSE types the client doesn't know about does not want to react too should just be silently discarded.

Comment (lolodomo) [reply bundles/org.openhab.binding.remoteopenhab/src/main/java/org/openhab/binding/remoteopenhab/internal/rest/RemoteopenhabRestClient.java:404]: It was to detect potential new messages types not yet supported by the binding. 
But yes, I could rather ignore them silently.

Comment (lolodomo) [reply bundles/org.openhab.binding.remoteopenhab/src/main/java/org/openhab/binding/remoteopenhab/internal/rest/RemoteopenhabRestClient.java:404]: Done

**Comment by J-N-K** [comment]
I belege this does not depend on the core PR but should be merged before, correct?

**Comment by lolodomo** [comment]
> I believe this does not depend on the core PR but should be merged before, correct?

As the current alive event has "event" as name, I believe you are right.

**Comment by lolodomo** [comment]
So, to be clear, this PR is working with and without the core PR changing the alive event.

**Comment by lolodomo** [comment]
Can this very small PR be merged please? 
@jlaur or @fwolter

**Review by jlaur — APPROVED** [review]
LGTM

**Pull request merged**